pkg/config/validation/validation.go

```diff
@@ -1960,10 +1975,175 @@ var ValidateVirtualService = registerValidateFunc("ValidateVirtualService",
 		return errs.Unwrap()
 	})
 
+func assignExactOrPrefix(exact, prefix string) string {
```

Comment:
it  seems exact and prefix can not be set at the same time, so the if/ else clause can be simplified

Reply:
it's done